dockers/base-xla/Dockerfile

```diff
@@ -92,6 +92,9 @@ RUN \
     python --version && \
     cd pytorch-lightning && \
     pip install -q fire && \
+    # Pin mkl version to avoid OSError on torch import
+    # OSError: libmkl_intel_lp64.so.1: cannot open shared object file: No such file or directory
+    pip install mkl==2021.4.0 && \
```

Comment:
@akihironitta Is there a reference to why we have to do this?

Reply:
@kaushikb11 Lost the link somewhere... Let me check and add it there.

Reply:
```suggestion
    # Pin mkl version to avoid OSError on torch import
    # OSError: libmkl_intel_lp64.so.1: cannot open shared object file: No such file or directory
    # https://github.com/pytorch/xla/issues/1666
    pip install mkl==2021.4.0 && \
```

Reply:
https://github.com/pytorch/xla/issues/1666

@kaushikb11 Please have a look again!

Reply:
Thanks @akihironitta! ❤️